src/DateUtils.ts

```diff
@@ -136,3 +136,29 @@ export function wantsDateSeparator(prevEventDate: Date, nextEventDate: Date): bo
     // Compare weekdays
     return prevEventDate.getDay() !== nextEventDate.getDay();
 }
+
+export function formatFullDateNoDay(date: Date) {
+    return (
+        date.getFullYear() +
+        "-" +
+        pad(date.getMonth() + 1) +
+        "-" +
+        pad(date.getDate()) +
+        _t(" at ") +
```

Comment:
this doesn't work for internationalisation as not all languages say things in the same order, you ought to use a formatting string like `_t("%(year)s-%(month)s-%(day)s at %(hours)s:%(minutes)s", { year, month, day, hours, minutes })`